README.md

```diff
@@ -7,11 +7,13 @@
 2. Extract to the root of your 3DS SD card
 3. Install the Nimbus homebrew using FBI (or FBI Reloaded) if using the CIA build
 4. Run the Nimbus homebrew and select either to use a Pretendo or Nintendo account
-     - If it doesn't work, reboot your 3DS while holding SELECT and ensure that "Enable loading external FIRMs and modules" as well as "Enable game patching" are both turned on
+     - If it doesn't work, reboot your 3DS while holding SELECT and ensure that "Enable loading external FIRMs and modules" is turned on, as well as ensuring that your Luma3DS version is 13.0 or higher.
```

Comment:
The `Enable game patching` option *is* required for the juxt patches, so that option should be left there

Everything else lgtm

Reply:
Fixed in https://github.com/TraceEntertains/nimbus/commit/83043a20f35aad22bd2f72da4791991914da5838